interface/src/app/mlmodel/mlModelComponents.js

```diff
@@ -30,4 +31,45 @@ angular.module('adage.mlmodel.components', [
   }]
 })
 
+.component('mlModelView', {
+  templateUrl: 'mlmodel/view.tpl.html',
+  controller: ['GlobalModelInfo', function(GlobalModelInfo) {
+    this.modelInfo = GlobalModelInfo;
+  }]
+})
+
+.component('mlModelValidator', {
```

Comment:
Also, I can't seem to find the mlmodel/validator.tpl.html file for this component - is it included in this PR?